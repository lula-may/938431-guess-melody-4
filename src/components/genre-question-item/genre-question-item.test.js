import React from "react";
import renderer from "react-test-renderer";
import GenreQuestionItem from "./genre-question-item.jsx";

const answer = {
  src: `https://samples/1.ogg`,
  genre: `blues`,
  id: `answer-3`
};

describe(`GenreQuestionItem`, () => {
  it(`should render correctly GenreQuestionItem component`, () => {
    const tree = renderer.create(
        <GenreQuestionItem
          answer={answer}
          id={0}
          onChange={() => {}}
          renderPlayer={() => {}}
          userAnswer={true}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

