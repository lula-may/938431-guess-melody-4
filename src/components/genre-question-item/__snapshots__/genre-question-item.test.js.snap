// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreQuestionItem should render correctly GenreQuestionItem component 1`] = `
<div
  className="track"
>
  <div
    className="game__answer"
  >
    <input
      checked={true}
      className="game__input visually-hidden"
      id="answer-3"
      name="answer"
      onChange={[Function]}
      type="checkbox"
      value="answer-3"
    />
    <label
      className="game__check"
      htmlFor="answer-3"
    >
      Отметить
    </label>
  </div>
</div>
`;
